pkg/controller/controller_test.go

```diff
@@ -130,6 +132,64 @@ func TestController(t *testing.T) {
 			pvcHasInUseErrors: true,
 			pvcInUse:          false,
 		},
+		// test cases with volume in use error handling disabled.
+		{
+			Name:                           "Feat disabled, Resize PVC, no FS resize, pvc-inuse with failedprecondition",
```

Comment:
very minor nit: can "feat disabled" be renamed to something like "with volume-in-use enabled" ?

Reply:
Done